frameworks/sveltekit/src/routes/data.json.ts

```diff
@@ -0,0 +1,51 @@
+import { Role, Prisma } from '@prisma/client'
+import { prisma } from "../prisma";
+/** @type {import('./data.json').RequestHandler} */
+export async function get({ }) {
+  const enumValue:Role|undefined = Role.ADMIN;
```

Comment:
Oh, wait, I understand why it's used here. Can we add a comment above to explain that this type annotation and, hence, the import must be preserved to test that importing this enum works?

Reply:
Hi Alex, surely I can add a comment. As you understand, since some setting of Svelte Kit this problem would be caused only when enums are imported as values and types at the same time, I put the type annotation so that it would keep causing even Kit or Vite's default setting had changed slightly.